cmd/go-canal/main.go

```diff
@@ -39,6 +43,8 @@ func main() {
 	cfg.Password = *password
 	cfg.Flavor = *flavor
 
+	cfg.ReadTimeout = *readTimeout * time.Second
```

Comment:
no need to multi time.Second 
you should use "60s" instead of 60 in the command flag.